Scope.Client/API/Features/Client.cs

```diff
@@ -0,0 +1,294 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="Client.cs" company="Scope SL">
+// Copyright (c) Scope SL. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Scope.Client.API.Features
+{
+    using System;
+    using Mirror.LiteNetLib4Mirror;
+    using RemoteAdmin;
+    using Scope.Client.API.Features.Packets;
+    using Scope.Client.Events.EventArgs;
+    using Steamworks;
+    using UnityEngine;
+    using UnityEngine.SceneManagement;
+
+    /// <summary>
+    /// Represents the client.
+    /// </summary>
+    public class Client
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Client"/> class.
+        /// </summary>
+        internal Client()
+        {
+            MainMenu = UnityEngine.Object.FindObjectOfType<NewMainMenu>();
+        }
+
+        /// <summary>
+        /// Gets the <see cref="Client"/> instance.
+        /// </summary>
+        public static Client Instance => Loader.BepInExLoader.Instance.Client;
+
+        /// <summary>
+        /// Gets the client <see cref="API.Features.Hardware"/>.
+        /// </summary>
+        public Hardware Hardware => new();
+
+        /// <summary>
+        /// Gets the current scene.
+        /// </summary>
+        public Scene CurrentScene => SceneManager.GetActiveScene();
+
+        /// <summary>
+        /// Gets a value indicating whether or not the client is connected.
```

Comment:
“Gets a value indicating whether or not the client is connected to a server” would be better.